src/hooks/AuditStatus/useAuditLogActions.ts

```diff
@@ -108,9 +115,10 @@ const useAuditLogActions = ({
         setCriteriaValidation(criteriaData);
       }
     };
-
-    fetchCheckPolygons();
-    fetchCriteriaValidation();
+    if (!reportEntityType) {
```

Comment:
coupled with the suggestion above to create new buttonToggle states (new members of `statusActionMap`), this variable wouldn't be needed because it can be a member of the passed in `buttonToggle` object.